crawl-ref/source/spl-cast.cc

```diff
@@ -1702,6 +1702,17 @@ static desc_filter _targeter_addl_desc(spell_type spell, int powc, spell_flags f
     return nullptr;
 }
 
+
+string target_desc(const monster_info& mi, spell_type spell)
+{
+    int powc = calc_spell_power(spell, true);
+    const int range = calc_spell_range(spell, powc, false);
+    unique_ptr<targeter> hitfunc = find_spell_targeter(spell, powc, range);
+    vector<string> d = _desc_hit_chance(mi, hitfunc.get());
+    return d.empty() ? "" : d[0];
+}
```

Comment:
This helper function required in spl-cast.cc because _desc_hit_chance is inaccessible outside this file.